OpenIPSL/Examples/Tutorial/Example_4/Instructions/package.mo

```diff
@@ -0,0 +1,491 @@
+within OpenIPSL.Examples.Tutorial.Example_4;
+package Instructions "Instructions"
+  extends Modelica.Icons.Information;
+  annotation (DocumentationClass=true, Documentation(info="<html>
+  <h4>Before You Start</h4>
+  <h5>Prerequisites</h5>
+  <ul>
+    <li>Dymola/OpenModelica installation.</li>
+    <li>
+      <a href=\"https://github.com/SanPen/GridCal\">GridCal</a> setup to generate the power flow solutions. An installation procedure with Miniconda is outlined <a href=\"https://github.com/ALSETLab/SMIB_Tutorial/blob/master/docs/gridcal_installation.md\">here</a>.
```

Comment:
```suggestion
      <a href=\"https://github.com/SanPen/GridCal\">GridCal</a> setup to generate the power flow solutions. An installation procedure with Miniconda is outlined in the <a href=\"https://github.com/ALSETLab/SMIB_Tutorial/blob/master/docs/gridcal_installation.md\">GridCal installation instructions of ALSETLab/SMIB_Tutorial</a>.
```
"here"-links are very bad style. You should always have descriptive self-explanatory links.

Reply:
Actually, aren't those instructions the ones provided in `InstallingGridCal`? And if yes then we should rather like to the local ones here.